### DISCUSSION TITLE
AI椅子

### DISCUSSION THREAD

**Discussion by Raywei0214** [1101-GS3073-C/Python-Community-GS3073-C#595] [General]
環保已成為現今全球趨勢，什麼都要環保，這項產品AI椅子我覺得非常炫泡，是用AI設計出來的，但價錢根本離譜，誰會去買一張15000的塑膠椅。15000都可以買一張小沙發了。還有詭異的10000元泰迪貝玩具燈，我只能說世界上有錢人不佔少數。
https://www.mithus.co/products/kartell-ai-chair

**Comment by lukiea** [comment]
ai 科技源自於懶惰 在這樣的世界裡一張椅子，有這不限的可能性，他賣的不是錢，是對於未來的一個展望

**Comment by awehao** [comment]
上面說的真好，確實因為人類的惰性，才發明這些方便的東西使生活更加輕鬆，不過利大於弊啦，出發點也是為了大家著想，不過我相信未來，這些東西的價格會越來越低

**Comment by bww0401** [comment]
賣的全是概念，並不是單單的塑膠椅而已，用藝術品的角度去看，這個價錢合理拉

**Comment by magicmicheal** [comment]
感覺現在需要客製化的東西，若是結合AI的話好像較容易地達到量產，靠這AI去偵測、記憶使用者的喜好，就能達成客製化了，要是商人能壓低成本的生產這類的產品，應概會是個很大的市場

**Comment by larry40305** [comment]
剛剛看到一個AI牙刷也是不便宜，我得說AI要跟日常用品結合還有很長的路要走。

**Comment by Xuewen628** [comment]
只能說這是設計的初期階段吧
很多產品剛出來的時候也都很貴業讓人找不出甚麼優點阿
其中需要人類許多的改善過程
我們該期待的是最後成熟的產品吧

**Comment by 0212Jerry** [comment]
AI 設計出來的東西就目前來講成本都非常高，因此產品也不會便宜。但也有可能是商人的噱頭。等到Ai 的技術成形後價格肯定可以降下來，屆時大家比較能明確的判定其CP 值

**Comment by Vincent20030314** [comment]
AI的設計感覺都要當藝術品看待，不然這錢感覺花了很不合理。
